src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -415,9 +415,9 @@ type MessageDbCategory<'event, 'state, 'context> internal (resolveInner, empty)
     inherit Equinox.Category<'event, 'state, 'context>(resolveInner, empty)
     new(context: MessageDbContext, codec: IEventCodec<_, _, 'context>, fold, initial,
         // For MessageDb, caching is less critical than it is for e.g. CosmosDB
```

Comment:
```
        // While not necessary to control costs, caching can improve the throughput of your application
        // a few times over, as such you should only skip it if you know what you're doing
```

Reply:
Threw it in, there's too much now though...